src/components/core/Button/RegisterButton/RegisterButton.tsx

```diff
@@ -1,28 +1,29 @@
-import { ButtonHTMLAttributes, FC, ReactElement } from "react";
+import Link, { LinkProps } from "next/link";
+import { FC, ReactElement } from "react";
 
 import { PlusIcon } from "@/components/icons";
 import { cn } from "@/utils/cn";
 
-export interface Props
-  extends Omit<ButtonHTMLAttributes<HTMLButtonElement>, "children"> {
-  icon?: ReactElement;
+export interface Props extends Pick<LinkProps, "href"> {
```

Comment:
P5

export 지워도 될 것 같습니다!

```suggestion
interface Props extends Pick<LinkProps, "href"> {
```